test/clj/nnts2/test_helpers.clj

```diff
@@ -0,0 +1,36 @@
+(ns nnts2.test-helpers
+  (:require  [clojure.test :as t]
+             [clojure.string :as str]
+             [org.httpkit.client :as http]
+             [nnts2.core :as core]
+             [cheshire.core :as json]))
+
+(defn settings
+  [setting]
+  (let [port 8000
+        host (str/join ["http://localhost:" port "/"])
+        base-url host]
+    (setting {:port port :host host :base-url base-url})))
+
+(defn http-request-raw
+  ([method url] (http-request-raw method url nil))
+  ([method url body]
+   (let [params   (merge {:url url
+                          :method method
+                          :headers  {"authorization" (first (str/split-lines (slurp "/Users/murtaza/Dev/nnts2/test/clj/nnts2/token")))
```

Comment:
Use a relative path here so that this line works across different setups

Reply:
Yeah, I meant to. I'll change it.